checador/db/migrate/20210723044029_create_companies.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCompanies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :companies do |t|
+      t.string :name
+      t.string :addres
```

Comment:
You have a `typo` here, I think you mean `address`

Reply:
Thanks, the changes were made, could you check